src/main/java/racinggame/Cars.java

```diff
@@ -0,0 +1,71 @@
+package racinggame;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import racinggame.Car.CarDto;
+
+public class Cars {
+
+    private static final int DEFAULT_POSITION = 1;
+    private List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public static Cars of(int count) {
+        if (!isPositive(count)) {
+            throw new IllegalArgumentException("음수를 사용 할 수 없습니다.");
+        }
+        return new Cars(carInitialization(count));
+    }
+
```

Comment:
Cars 생성자에서 비어있는 자동차를 체크해도 되지 않을까 싶어요
```suggestion
    public Cars(List<Car> cars) {
        if (cars.isEmpty()) {
            // 예외
        }
        this.cars = cars;
    }

    public static Cars of(int count) {
        return new Cars(carInitialization(count));
    }

```